easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-foss-2017b-Python-3.6.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+libpng_ver = '1.6.32'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '3.6.2'),
+    ('libpng', libpng_ver),
+    ('freetype', '2.7.1', '-libpng-%s' % libpng_ver, ('GCCcore', '6.4.0')),
```

Comment:
@schiotz Why not use the existing easyconfig for `freetype` 2.8?

Also, no need to hardcode the subtoolchain, so just use `('freetype', '2.8')` (and remove the `freetype` easyconfig you added)?

Reply:
I thought so, too.  But it did not work, perhaps because the freetype .eb file was not in the repository yet.

Reply:
If you point `--robot` to the location of the `freetype` easyconfig, that should work just fine.